### DISCUSSION TITLE
Code review ІМ-12

### DISCUSSION THREAD

**Discussion by MrSampy** [HowProgrammingWorks/LiveQA#51] [Code review]
Пишу генератор карти.
Генератор спочатку створює карту висот, а згідно данних першої створює карти тепла, (у майботньому ще додам вологості та біомів). Також хочу ще додати генерацію річок та  інтерфейс, щоб користувач міг вибрати, який вид карти він хоче створити.
Опис поки не робив. Буду вдячній за ревью [кода](https://github.com/MrSampy/Course-work).

**Comment by tshemsedinov** [comment]
https://github.com/MrSampy/Course-work/issues/1

**Reply by MrSampy** [reply]
All fixed
